### DISCUSSION TITLE
Arcade Games e Daphne not work for me please HELP ME in RG350p

### DISCUSSION THREAD

**Discussion by npjunior3d** [eduardofilo/RG350_adam_image#247] [Q&A]
hello, guys, how are you... very cool the project i'm having some problems here... i'm using fw on an RG350P i put some games on 64gb sd card, but i couldn't make daphne games run if anyone can help me how to do it would be very useful. And the MAME games I put in the arcade folder but it doesn't find any of them. By any chance in FW is there a way to force him to scan the list of games or manually put the path of roms that he did not find on the SD card?
grateful for the attention

**Comment by neilswann80** [comment]
Are the MAME games .zip files?

**Reply by npjunior3d** [reply]
yes

**Reply by neilswann80** [reply]
That's strange, SimpleMenu should automatically list all .zip files from the external sdcard directory `/roms/ARCADE` in the Arcades/Mame section

There's nothing unusual about the naming of the roms?  Do they become visible by disabling "Tidy rom names" (from the "start" menu) within SimpleMenu?

What version of Adam Image are you using?

**Reply by eduardofilo** [reply]
Have you used SimpleMenu navigation as described in this question?
https://github.com/eduardofilo/RG350_adam_image/wiki/En:-7.-FAQ#q10-why-dont-appear-in-simplemenu-the-roms-i-just-loaded-onto-the-external-card

**Reply by eduardofilo** [reply]
Is your card formatted in FAT32 as described here?
https://github.com/eduardofilo/RG350_adam_image/wiki/En:-3.-Content-installation#external-microsd-format-and-label

Is directory scaffolding generated?

**Reply by npjunior3d** [reply]
Yes i format in fat32 and use guiformat o try fins simple menu

**Reply by neilswann80** [reply]
Have you got a spare sdcard you could wipe and format as FAT32?  Whilst the card is still empty, before putting any games on it,  boot it up in your RG350P, immediately shut down the device, remove the sdcard and connect to your PC.  Confirm the folder structure has been created on the sdcard, then transfer a few MAME games to `/roms/ARCADE` then reinsert in your device, boot and test again.

**Reply by eduardofilo** [reply]
> Yes i format in fat32 and use guiformat o try fins simple menu

In that case, make a video exactly like this one. Alternatively, snapshots of the key screens may serve.
https://www.youtube.com/watch?v=N6mw0L3dU9w

**Reply by npjunior3d** [reply]
ok i try this i use [v1.4.2] ok

**Reply by npjunior3d** [reply]
https://youtu.be/C3oyEjdrG3k

**Comment by npjunior3d** [comment]
https://youtu.be/C3oyEjdrG3k



Em ter., 3 de mai. de 2022 18:57, Eduardo Moreno ***@***.***>
escreveu:

**Reply by neilswann80** [reply]
They are `.rar` compressed files.  If you unpack them and repack as `.zip` instead they should work.

This [POST](https://superuser.com/questions/14946/is-there-a-simple-automated-way-to-convert-a-large-number-of-files-between-zip) suggests you should be able to bulk-covert the `.rar` files to `.zip` using WinRAR.

FYI... `.rar` compressed files aren't listed as supported by any of the emulators within Adam Image; DAPHNE roms should also be `.zip`

**Reply by npjunior3d** [reply]
ok sorry to bother you but wouldn't you have an image of your game card with the games mom and daphne already configured? why i believe this must be the error or in the bios that are in different version. I noticed when I used the scrapper program it renamed the rom to .rar I put it back in .zip they appeared but no rom opens. thanks for listening

**Reply by neilswann80** [reply]
Did you just rename the files or properly unpack and repack in the correct format?  You have to do the latter, a simple rename will not suffice.

Sharing roms is still very much a legal grey area so I wouldn't encourage that option.  The best solution would be to get your roms back into the correct format.

**Reply by npjunior3d** [reply]
good morning, guys after a lot of struggle I found out supposedly that the error in the roms occurs when I use the scrapper to download the previews.
I need a link where I can download daphne that works I've already downloaded it more than 3 places they seem to me for PC and they don't run on linux by the way could someone get me?

**Reply by neilswann80** [reply]
Strange... I used [SKRAPER](https://www.skraper.net/) and didn't get the issue you're reporting.  

I've tested "Dragon's lair" 1 & 2 which both worked on Adam Image for me.

HINT: You might be able to obtain the same collection of roms containing the games I tested by using search terms like:
`Ultimate Daphne ROMs Set`

(How you obtain the roms is your decision and you should ensure it's legal for you to acquire them)

**Comment by npjunior3d** [comment]
one more doubt in the ADAM Image I do not have access to Openbor games? Any way to make him appear too?

**Reply by neilswann80** [reply]
Reading [HERE](https://boards.dingoonity.org/retro-game-350rg-350/rg350-openbor/) it suggests not all games are going to work on the port of OpenBor for OpenDingux.

I'm not sure whether OpenBor can handle compressed archive files, but it's probably better to unpack the PAK files into:  `/media/sdcard/OpenBOR/Paks`

**Reply by eduardofilo** [reply]
You need to install the emulator that is not included in Adam image by default. You have it listed in the list of game ports curated by Neil in 118 position:
https://github.com/eduardofilo/RG350_adam_image/wiki/En:-8.-Compatible-game-ports

**Reply by neilswann80** [reply]
Ah yes, sorry.  I had presumed that step had already occurred... but possibly not!
